cli/tests/integrations/test_help.py

```diff
@@ -34,6 +34,7 @@ def test_list():
 \tpackage        \tInstall and manage DCOS software packages
 \tservice        \tManage DCOS services
 \ttask           \tManage DCOS tasks
+\ttunnel         \tRun a tunnel through DCOS cluster nodes
```

Comment:
None really, now that you mention it, it'd require alot fewer changes if I moved it into a node subcommand.